src/main/resources/default/taglib/w/multiLanguageTextField.html.pasta

```diff
@@ -9,90 +9,107 @@
+            </div>
+            <div class="mls-modal-inputs">
 
-        </div>
-    </w:modalBody>
-    <w:modalFooter>
-        <div class="mls-add-language-button btn-group dropup pull-left hidden">
-            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown"
-                    aria-haspopup="true" aria-expanded="false">
-                <i class="fa fa-plus"></i> @i18n("MultiLanguageEditor.addLanguage")
-            </button>
-            <ul class="dropdown-menu"></ul>
-        </div>
+            </div>
+        </w:modalBody>
+        <w:modalFooter>
+            <div class="mls-add-language-button btn-group dropup pull-left hidden">
+                <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="false">
+                    <i class="fa fa-plus"></i> @i18n("MultiLanguageEditor.addLanguage")
+                </button>
+                <ul class="dropdown-menu"></ul>
+            </div>
 
-        <a class="btn btn-primary" data-dismiss="modal">@i18n("NLS.close")</a>
-    </w:modalFooter>
-</w:modal>
+            <a class="btn btn-primary" data-dismiss="modal">@i18n("NLS.close")</a>
+        </w:modalFooter>
+    </w:modal>
 
-<script>
-    $(document).ready(function () {
-        let validLanguages = {};
-        <i:for type="List" var="lang" items="validLanguages">
-            validLanguages['@lang'] = '@i18n("Language." + lang)';
-        </i:for>
+    <script>
+        $(document).ready(function () {
+            let validLanguages = {};
+            <i:for type="List" var="lang" items="validLanguages">
+                validLanguages['@lang'] = '@i18n("Language." + lang)';
+            </i:for>
 
-        let field = new MultiLanguageField({
-            fieldName: '___fieldName',
-            hasFallback: ___value.isWithFallback(),
-            fallbackLabel: '@i18n("Language.fallback")',
-            wrapperId: '___fieldId',
-            modalId: '___modalId',
-            values: JSON.parse(<i:raw>'___value.getAsJSON()'</i:raw>),
-            defaultLanguage: '___defaultLanguage',
-            validLanguages: validLanguages,
-            languageManagementEnabled: ___languageManagementEnabled,
+            let field = new MultiLanguageField({
+                fieldName: '___fieldName',
+                hasFallback: ___value.isWithFallback(),
+                fallbackLabel: '@i18n("Language.fallback")',
+                wrapperId: '___fieldId',
+                modalId: '___modalId',
+                values: JSON.parse(<i:raw>'___value.getAsJSON()'</i:raw>),
+                defaultLanguage: '___defaultLanguage',
+                validLanguages: validLanguages,
+                languageManagementEnabled: ___languageManagementEnabled,
+            });
         });
-    });
-</script>
+    </script>
+
+    <i:else>
+        <w:textfield span="@span"
```

Comment:
do i understand this codechange correctly that it now is no longer possible to see other language values when the field is readonly? I think the readonly flag should rather be passed to the  js MultiLanguageField and handled in the created textareas and modal.

Reply:
right